packages/@misk/cli/src/utils/generate.ts

```diff
@@ -10,21 +10,33 @@ import {
 } from "./templates"
 import { getVersion } from "./changelog"
 
-export const generateBuildFiles = async () => {
-  const pkg = await fs.readJson(Files.package)
-  const miskTab: IMiskTabJSON = await fs.readJson(Files.miskTab)
+export const generateBuildFiles = async (...args: any) => {
+  const dir = args[0]
+  let pkg = {}
+  try {
+    pkg = await fs.readJson(`${dir}/${Files.package}`)
+  } catch (e) {}
```

Comment:
? Why are we ignoring exceptions here?

Reply:
synced up. We're using exception based control flow here, which is generally considered an anti-pattern. Let's find the proper "does file exist" API to use here that doesn't throw an exception if the file does not exist.